src/plugins/expressions/common/execution/execution.ts

```diff
@@ -489,10 +489,6 @@ export class Execution<
           continue;
         }
 
-        if (!aliases?.length) {
-          throw new Error(`${fnDef.name} requires an argument`);
-        }
-
         // use an alias if _ is the missing arg
         const errorArg = name === '_' ? aliases[0] : name;
```

Comment:
this line has a potential to throw (when aliases is not an array).

`        const errorArg = name === '_' && aliases?.length ? aliases[0] : name;`

Reply:
https://github.com/elastic/kibana/blob/e8f59a588886f10eb4d37bfd80742a9239190ee9/src/plugins/expressions/common/expression_functions/expression_function_parameter.ts#L30

Reply:
in that case we should get rid of `errorArg` and just use `name`

Reply:
Checked everything. Works as expected. That check was not used.